schemas/common/email-interaction.schema.json

```diff
@@ -0,0 +1,136 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/emailinteraction",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Email Interaction",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Captures information in the context of an email correspondence between persons",
+  "definitions": {
+    "emailinteraction": {
+      "properties": {
+        "xdm:priority": {
+          "title": "Email's priority",
+          "type": "string",
+          "description": "Captures the priority set by the person who sent the email"
+        },
+        "xdm:sentDateTime": {
```

Comment:
can just be `xdm:sentDate`

Reply:
Given it has a "time" component to the value "Date and time the ...". I think sendDateTime makes sense here.

Reply:
We don't do that for `createDate`, `modDate`, etc. - why this one?

Reply:
The only other "createDate" we have is from "repo:createDate". Not sure we need an "xdm:createDate" and "repo:createDate". If we are really creating a new "xdm" name spaced date field then I like "Time" in the name to help it match its specific format requirements. I'll let @mjasrotia and @prabhum2 make the final call here.

Reply:
This is a valid point. @prabhum2  and @mjasrotia  - You recently created an xdm:createDate in the "source system audit" schema which is also date-time. Seems like we should be consistent.